docs/Makefile

```diff
@@ -19,7 +19,12 @@ docs-debug: docs docs-image
 		--rm \
 		--name vmdocs-docker-container \
 		-p 1313:1313 \
-		-v ./docs:/opt/docs/content/victoriametrics vmdocs-docker-package
+		-v ./docs/anomaly-detection:/opt/docs/content/victoriametrics \
+		-v ./docs/guides:/opt/docs/content/guides \
+		-v ./docs/victoriametrics:/opt/docs/content/victoriametrics \
+		-v ./docs/victoriametrics-cloud:/opt/docs/content/victoriametrics-cloud \
+		-v ./docs/victorialogs:/opt/docs/content/victorialogs \
+			vmdocs-docker-package
```

Comment:
running `docs-debug` seems broken: i see no victorialogs, guides, cloud and other docs in renderred result

Reply:
this change doesn't work without changes in vmdocs